### PR TITLE
read in existing auxiliary files into msh obj, updates to make_bc

### PR DESCRIPTION
**Pull request by WPringle**
Some additions for sponge zone construction, reading auxiliary files, writing offset63, swanoutput namelist, update Make_f5354 help and min_depth option.

### CODE REVIEW

**Comment by WPringle** [comment]
Sponge example:

```
% input files
mesh_file = 'my_mesh.mat'
tpxoh = 'h_**_tpxo9_atlas_30_v5.nc';
tpxou = 'u_**_tpxo9_atlas_30_v5.nc';

% load the mesh
load(mesh_file)

% add the sponge layer (width 10% of M2) 
sponge_opt = 1; % for tidal boundary conditions
m = Calc_Sponge(m,[12.42*3600, 0.1],sponge_opt);

%% Make the fort.15 struct
TS = start_time; %[date-string]
TE = end_time; %[date-string]
DT = time_step; %[s]
CONST = "major8"; % tidal constituent information
m = Make_f15(m,TS,TE,DT,'const',CONST,'tidal_database',tpxoh);

% add the sponge info for this tidal
m = Make_f5354(m, tpxoh, tpxou);
% write out mesh files..
write(m,'sponge_mesh')
```

File: README.md [hunk 1]
```diff
@@ -157,9 +157,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ### Unreleased (on current HEAD of the Projection branch)
 ## Added
-
-- Added new function in `msh` called `remesh_patch` to remesh within specified polygonal domains and insert back into parent mesh.
```
Comment (krober10nd) [inline README.md:161]: Please preserve the prior recently added items on the checklist.

**Review by krober10nd — APPROVED** [review]
I like the ability to read in aux files using the `msh.read()` perhaps this could be added to the README as a code snippet and a short couple sentences?

**Comment by krober10nd** [comment]
The only other comment pending is regarding preserving the previous entries in the changelog. Otherwise good to go.

**Pull request merged**